src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T & string): GroupsMap<T> {
```

Comment:
![image](https://user-images.githubusercontent.com/122989585/223804390-816b7c7b-d078-4a9a-968a-59706e19a2f0.png)
```suggestion
export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
```